### DISCUSSION TITLE
Doubts about payments split and user roles

### DISCUSSION THREAD

**Discussion by SrLucca** [pretix/pretix#3820] [Q&A]
I wanna to use petrix like an Saas, but i have some question about how can i do split payment and roles that users can have.

1 - Is it possible to split payments using Stripe, and does the platform have a configuration for this?

2 - Is it possible to set a global rate that will be applied to any ticket sales by the organizers?

3 - How refunds are implemented on the platform and whether any fees apply, as payment getaways generally charge.

4 - Is it possible for any user to create a registration on the platform and have different roles?

If anyone can explain me, thank you very much.

**Comment by raphaelm** [comment]
There is limited support for Stripe application fees in pretix, but it's really not something pretix is built for. I don't even know how refunds are handled, and it's not something we are actively supporting.
